test/ec_internal_test.c

```diff
@@ -433,6 +434,73 @@ int ecpkparams_i2d2i_test(int n)
     return testresult;
 }
 
+
+static int check_bn_mont_ctx(BN_MONT_CTX *mont, BIGNUM *mod, BN_CTX *ctx)
+{
+    int ret = 0;
+    BN_MONT_CTX *regenerated = BN_MONT_CTX_new();
+
+    if (!TEST_ptr(regenerated))
+        return ret;
+    if (!TEST_ptr(mont))
+        goto err;
+    
+    if (!BN_MONT_CTX_set(regenerated, mod, ctx)) {
```

Comment:
Use the test macros..

Remove the TEST_error() lines and do the following..
````
if (!TEST_true())
     goto err;
````